Google.Api.Generator/Utils/Disposable.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Google.Api.Generator.Utils
+{
+    internal class Disposable : IDisposable
```

Comment:
I think a comment would be useful on this class, mostly to give an idea of when it's useful.

Reply:
Done